### DISCUSSION TITLE
Add New game Palworld

### DISCUSSION THREAD

**Discussion by Colonel-Hannib** [GameServerManagers/LinuxGSM#4452] [Ideas]
Hello 

There's a new game out this week: Palworld.
I found the documentation to create dedicated servers.

https://tech.palworldgame.com/dedicated-server-guide#linux

Do you think it's possible to add it?

Sincerely
Guillaume

**Comment by PlayMTL** [comment]
Palworld Server was released in https://github.com/GameServerManagers/LinuxGSM/releases/tag/v24.1.0
